obj.go

```diff
@@ -107,6 +127,20 @@ func (O *Object) Get(name string) (interface{}, error) {
 	return sub, nil
 }
 
+// ObjectRef implements userType interface.
+func (O *Object) ObjectRef() *Object {
+	return O
+}
+
+// Close releases a reference to the object.
+func (O *Object) Close() error {
+	if rc := C.dpiObject_release(O.dpiObject); rc == C.DPI_FAILURE {
```

Comment:
Shouldn't we mark somehow that this Object has been closed? 
Like setting O.dpiObject to nil ?

Reply:
Ok